k8s/perftest/common/draft-store/draft-store-service.yaml

```
@@ -0,0 +1,45 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: draft-store-service
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
```

this is the wrong helm chart repo.
```suggestion
    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
```